### PR TITLE
Updated the packs category to *Authentication & Identity Management*  (part 2)

### PR DESCRIPTION
**Pull request by maimorag**
## Status
- [x] In Progress

## Related Issues
relates: [link to the issue](https://jira-hq.paloaltonetworks.local/browse/CIAC-5348)

## Description
_**part two:**_
Change the pack category `Authentication` to `Authentication & Identity Management`. e.g packs: Okta, Azure AD, Ping Identity.

## Minimum version of Cortex XSOAR
- [x] 6.0.0

### CODE REVIEW

File: Packs/Attlasian/Integrations/Attlasian_IAM/Attlasian_IAM.py [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 import urllib3
 # Disable insecure warnings
+import urllib3
```
Comment (Shellyber) [inline Packs/Attlasian/Integrations/Attlasian_IAM/Attlasian_IAM.py:7]: duplicate, no?

Comment (maimorag) [reply Packs/Attlasian/Integrations/Attlasian_IAM/Attlasian_IAM.py:6]: yes fixed

**Review by Shellyber — APPROVED** [review]
Nice.

**Comment by maimorag** [comment]
The following validation error occurs since Google-Vault-Generic-Test is skipped (Issue 24347):

Packs/GoogleVault/Integrations/GoogleVault/GoogleVault.yml - [CJ105]
 
 @DeanArbel can you please force-merge?

**Pull request merged**